include/nn/pdm/pdm_cpp.h

```diff
@@ -0,0 +1,466 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup nn_pdm
+ * \ingroup nn_pdm
+ * Log Activities for Titles
+ * @{
+ */
```

Comment:
Doxygen considers C++ functions separately to C functions, putting them in their own area called "Namespaces" - it'd be good to include the text `nn::pdm` here (or like.. "`see nn::pdm`") so it'll generate a link to the Namespace area. An example of this is [nn_ac](https://wut.devkitpro.org/group__nn__ac__cpp.html) - if you don't use the nn::ac link in the text and instead click "nn", it'll show you all the libraries. It's a bit weird and we probably need to change it, but hey, it's best practice for now.